locales/he_IL/LC_MESSAGES/duckduckgo.po

```diff
@@ -99,14 +101,16 @@ msgid ""
 "%sClick here%s to try again, if you think there should be results for this "
 "search."
 msgstr ""
+"%sלחץ כאן%s כדי לנסות שוב, if you think there should be results for this "
+"search."
```

Comment:
This is the string I wanted you to double check. As you can see, it is only partially translated.